h2o-algos/src/main/java/hex/tree/SharedTree.java

```diff
@@ -791,7 +791,7 @@ static protected void printGenerateTrees(DTree[] trees) {
           writer.println(dtree.root().toString2(new StringBuilder(), 0));
           writer.close();
         } catch (FileNotFoundException|UnsupportedEncodingException e) {
-          e.printStackTrace();
+          throw new RuntimeException(e);
```

Comment:
I think we can just delete this whole method

maybe we can add `toDebugString()` to `DTree` - it could do `dtree.root().toString2(new StringBuilder(), 0)` instead